### ISSUE TITLE
Package integration is broken after Angular CDK 14.1.0

### ISSUE THREAD

**Issue by Sergiobop** [MurhafSousli/ngx-scrollbar#448]
After updating CDK to 14.1.0, the integration with cdk-virtual-scroll-viewport is acting weird, adding spaces and scroll when is not neccesary.
Reverting to 14.0.6 fixes the issue.

I think this PR can be related https://github.com/angular/components/pull/24394

Stackblitz:
https://ngx-scrollbar-c3u9al.stackblitz.io

As you can see, i only updated CDK and Material to 14.1.0. In the virtual scroll example, you can see a 'cdk-virtual-scroll-spacer' with height: 50000px (an enormous blank space) after the elements:
<img width="367" alt="Captura de Pantalla 2022-07-20 a las 21 08 25" src="https://user-images.githubusercontent.com/73531831/180062686-e9912fae-0f95-482f-99ed-a5d44bc54c94.png">

**Comment by MurhafSousli** [comment]
I think you shared the wrong link, I don't the issue there!

Please check it again, and send me the editable link, not only the preview.

https://stackblitz.com/edit/ngx-scrollbar-c3u9al?file=src%2Fapp%2Fvirtual-scroll-example%2Fvirtual-scroll-example.html

**Comment by Sergiobop** [comment]
Oh sorry, in the link you provided, you can already see the problem:
https://stackblitz.com/edit/ngx-scrollbar-c3u9al?file=src%2Fapp%2Fvirtual-scroll-example%2Fvirtual-scroll-example.html

**Comment by NimeTop** [comment]
You got any chance to look into this one @MurhafSousli ? Thanks!

**Comment by MurhafSousli** [comment]
I figure out what went wrong! basically in Angular CDK **v14.1.0**, they [modified some of the styles](https://github.com/angular/components/pull/24394/files#diff-788bb14508b22c01b91e2f47d9ee53f3d0170ecacd9fb31c60dba64998912a23) which caused this bug


Before **v14.1.0** both `.cdk-virtual-scroll-content-wrapper` and `.cdk-virtual-scroll-spacer` had absolute position. However, to make our custom scrollbar library work we had to override `.cdk-virtual-scroll-content-wrapper` position to relative. _this worked fine since the spacer is still absolute_.

**The problem** is when they removed `position: absolute` from the spacer element in **v14.1.0**, the space went down under the `.cdk-virtual-scroll-content-wrapper` because both elements have relative positioning hence the bug was introduced.

There are two possible fixes:

1. Remove our override of `position: relative !important`  in `.cdk-virtual-scroll-content-wrapper`
2. Override `.cdk-virtual-scroll-spacer` position to absolute

___

**The first possible fix is not an option!**

![image](https://user-images.githubusercontent.com/8130692/183270323-ed4c1db5-5c0b-41e2-9262-2226e7b930f8.png)

We cannot remove that override of `.cdk-virtual-scroll-content-wrapper` because it will introduce another bug related to the horizontal scrollbar.

![image](https://user-images.githubusercontent.com/8130692/183270348-b43bd2f3-e7a8-4d31-899f-567d678fd583.png)

___

The second possible fix is to bring back the absolute positioning on `.cdk-virtual-scroll-spacer`

```scss
.cdk-virtual-scroll-spacer {
    position: absolute;
    top: 0;
    left: 0;
    right: 0;
}
```

I really don't like to name 3rd party library classes in this library to keep it independent. but maybe we don't have many options.

**Comment by MurhafSousli** [comment]
Fixed in v14.1.0
